art-feature.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Art Feature</title>
+    <link rel="icon" href="/img/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+  <body>
+    <h1 class="welcome-title">Your Art</h1>
+    <div class="editor">
+      <div class="html-container">
+        <input type="text" placeholder="Thinking is the enemy of creativity." />
+        <textarea id="html-code"></textarea>
+      </div>
+      <div class="output-container">
+        <iframe id="output"></iframe>
+      </div>
+    </div>
+    <div class="container">
+      <button class="button">Submit for 500 Dineros</button>
```

Comment:
What is the purpose of the button here... I have not observed any ID attached to it or any function being called onclick of this... Please point me out to the code if I had missed it

Reply:
The purpose of this button is not yet defined but I am sure about it but it's something related to the backend of this feature😅